module main;


class TestException : int
{
	let Message (- string;
}

def main()
{
	;
}